base/linalg/cholesky.jl

```diff
@@ -460,6 +460,7 @@ function A_ldiv_B!(C::CholeskyPivoted, B::StridedMatrix)
     end
 end
 
+isposdef(C::CholeskyPivoted) = C.rank > 0
```

Comment:
Restricting this method to LAPACK/BLAS eltypes might be wise, in case a generic pivoted Cholesky implementation that behaves differently comes along?

Reply:
I think the generic version should have the same behavior